### ISSUE TITLE
service/dap: `dlv config <param>` should print, not reset value without an explicit parameter

### ISSUE THREAD

**Issue by polinasok** [go-delve/delve#2890]
A very intuitive way to look up a config value is to run `dlv config <param>`. This worked as expected with  dlv@v1.7.3.
![image](https://user-images.githubusercontent.com/51177946/152398688-0d9349ef-a905-4136-870d-b163dfb7abad.png)
But with dlv@v1.8.0 and dlv@latest the same command tries to reset the value, which works only in some cases
![image](https://user-images.githubusercontent.com/51177946/152399014-11460f5b-0061-4485-9b1c-14440f5f10f8.png)
I think we should go back to previous consistent and intuitive behavior. Resetting values should be an explicit action. We don't want users to accidentally reset anything when they try to simply look up what they are or remind themselves what the syntax is. And there should be an easy way to look up an existing setting without having to dig through the full list of configuration values with `dlv config -list`.

@suzmue

**Comment by polinasok** [comment]
I just remember that this now works:
```
dlv config -list showGlobalVariables
showGlobalVariables	false
```
It's not ideal because `-list` appears before the parameter, so switching back and force between setting and looking up values requires changes in the middle of the command, which can get quite tedious.

**Comment by aarzilli** [comment]
Fixed by 13ad7dc1d581b86d7000f6fc4bd37309c30ec480
